mylar/PostProcessor.py

```diff
@@ -440,11 +440,11 @@ def Process(self):
                         return self.queue.put(self.valreturn)
                     logger.info('I have located %s files that I should be able to post-process. Continuing...' % filelist['comiccount'])
                 else:
-                    if all([self.comicid is None, '_' not in self.issueid]):
+                    if all([self.comicid is None, self.issueid is not None and '_' not in self.issueid]):
```

Comment:
Would it be clearer to have a variable encompass all the conditional logic amking it easier to read?

```suggestion
                    _is_comic_info_none = all([self.comicid is None, self.issueid is not None and '_' not in self.issueid])
                    if _is_comic_info_none:
```